src/client/components/MealListItem.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const MealListItem = ({ item }) => {
+  return (
```

Comment:
You could use object destructuring here and have 
`const {title, description, price} = item;
`Then you can use `title, description, price` directly 👍🏽

Reply:
Yes, I did that in todo homework.
Thank you for review, btw :)